src/main/java/seedu/exercise/ui/ResourceListPanel.java

```diff
@@ -93,17 +96,22 @@ private void setResourceListListener() {
             @Override
             public void onChanged(Change<? extends Resource> change) {
                 int index = -1;
+                Logger logger = LogsCenter.getLogger(ResourceListPanel.class);
                 while (change.next()) {
                     if (change.wasReplaced()) {
+                        logger.info("Was replaced" + index);
                         index = change.getFrom();
                         break;
                     } else if (change.wasAdded()) {
+                        logger.info("Was added" + index);
                         index = change.getFrom();
                         break;
                     } else if (change.wasRemoved()) {
+                        logger.info("Was removed" + index);
                         index = change.getFrom();
                         break;
                     } else if (change.wasUpdated()) {
+                        logger.info("Was updated" + index);
```

Comment:
Are you sure you want to keep the logging here? Clearly remembered it was for debugging purposes. What would be the purpose of the logs if you do keep it and if you do keep it, can change the level of logging to debug

Reply:
Will remove it. Thanks